### DISCUSSION TITLE
Add more Collaborators

### DISCUSSION THREAD

**Discussion by karl-police** [Epix-Incorporated/Adonis#1213] [General]
My last post was unethically locked https://github.com/Epix-Incorporated/Adonis/discussions/1194

This is a new one :)

I think there were 1 or 2 new collaborators, but we need more 😭

**Comment by Dimenpsyonal** [comment]
two? since when

**Comment by Expertcoderz** [comment]
Persistently begging to become a collaborator will, at best, decrease your future chances of actually becoming one.
